.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,79 @@
+  sonarcloud-analysis:
+    name: Analyze .NET Codebase with SonarCloud
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 🔄 Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: 🧰 Setup .NET SDK
+        uses: actions/setup-dotnet@v3
```

actionlint (1.7.4)

the runner of "actions/setup-dotnet@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)